Go Port Scanner Syntax Errors

# Description 
The Go port scanner returns syntax error when using the instructions to execute with go version `go1.16.3 linux/amd64`

**To Reproduce**
1. `$ wget https://raw.githubusercontent.com/HarshCasper/Rotten-Scripts/master/Go/Port_Scanner/port_scanner.go`
2. `go run ./port_scanner.go`
3. Syntax error:
```
$ go run ./port_scanner.go
# command-line-arguments
./port_scanner.go:39:17: syntax error: unexpected :, expecting {
./port_scanner.go:41:15: syntax error: unexpected {, expecting for loop condition
./port_scanner.go:42:19: syntax error: missing statement after label
./port_scanner.go:46:16: syntax error: unexpected :, expecting := or = or comma
./port_scanner.go:52:9: syntax error: non-declaration statement outside function body
./port_scanner.go:61:10: syntax error: missing statement after label
```
**Expected behavior**
Successful execution of the port scanner file

**Additional context**
System: Linux
Go Version: 1.16.3

--- a/Go/Port_Scanner/port_scanner.go
+++ b/Go/Port_Scanner/port_scanner.go
@@ -1,72 +1,69 @@
 package main
 
 import (
-    "fmt"
-    "net"
-    "time"
-    "sync"
-    "strings"
-
+	"fmt"
+	"net"
+	"strings"
+	"sync"
+	"time"
 )
 
-
 var wg sync.WaitGroup
+
 func main() {
 
-    var option int
-    var ip string
-    fmt.Print("[+] Enter IP address to scan: ")
-    fmt.Scan( & ip)
+	var option int
+	var ip string
+	fmt.Print("[+] Enter IP address to scan: ")
+	fmt.Scan(&ip)
 
-    var typeOfPort string
-    fmt.Print("[+] Enter type of port you want to scan for eg(tcp,udp..etc): ")
-    fmt.Scan( & typeOfPort)
+	var typeOfPort string
+	fmt.Print("[+] Enter type of port you want to scan for eg(tcp,udp..etc): ")
+	fmt.Scan(&typeOfPort)
 
-    fmt.Print("[+] For scanning all open ports press : 1 \n[+]For scanning Specific ports press :2 \n")
-    fmt.Scan( & option)
-    if option == 1 {
-        scanAllPorts(ip, typeOfPort)
-    } else if option == 2 {
-        scanSelectedPorts(ip, typeOfPort)
-    } else {
-        fmt.Println("Please chose valid option")
-    }
+	fmt.Print("[+] For scanning all open ports press : 1 \n[+]For scanning Specific ports press :2 \n")
+	fmt.Scan(&option)
+	if option == 1 {
+		scanAllPorts(ip, typeOfPort)
+	} else if option == 2 {
+		scanSelectedPorts(ip, typeOfPort)
+	} else {
+		fmt.Println("Please chose valid option")
+	}
 }
 
 // function for scanning all Ports
 func scanAllPorts(ip string, typeOfPort string) {
-        wg.Add(65535)
-        for port: = 1;
-        port < 65535;
-        port++{
-            hostIp: = fmt.Sprintf("%s:%d", ip, port)
-                // fmt.Println(hostIp)
+	wg.Add(65535)
+	for port := 1; port < 65535; port++ {
+		hostIp := fmt.Sprintf("%s:%d", ip, port)
+		// fmt.Println(hostIp)
 
-                _,
-            err: = net.DialTimeout(typeOfPort, hostIp, 10 * time.Millisecond)
+		_, err := net.DialTimeout(typeOfPort, hostIp, 10*time.Millisecond)
 
-            if err == nil {
-                fmt.Printf("Port %d is open \n", port)
-            }
-        }
-        wg.Wait()
+		if err == nil {
+			fmt.Printf("Port %d is open \n", port)
+		}
+	}
+	wg.Wait()
 
-        wg.Done()
-    }
-    // function for scanning specific Ports
+	wg.Done()
+}
+
+// function for scanning specific Ports
 func scanSelectedPorts(ip string, typeOfPort string) {
-    var portsInput string
-    fmt.Print("[+] Enter Ports to scan: ")
-    fmt.Scan( & portsInput)
-    ports: = strings.Split(portsInput, ",")
-    for _, port: = range ports {
-        hostIp: = fmt.Sprintf("%s:%s", ip, port)
-        _,
-        err: = net.DialTimeout(typeOfPort, hostIp, 80 * time.Millisecond)
-        if err != nil {
-            fmt.Printf("Port %s is closed \n", port)
-        } else {
-            fmt.Printf("Port %s is open \n", port)
-        }
-    }
+	var portsInput string
+	fmt.Print("[+] Enter Ports to scan: ")
+	fmt.Scan(&portsInput)
+	ports := strings.Split(portsInput, ",")
+	for _, port := range ports {
+		hostIp := fmt.Sprintf("%s:%s", ip, port)
+
+		_, err := net.DialTimeout(typeOfPort, hostIp, 80*time.Millisecond)
+		if err != nil {
+			fmt.Printf("Port %s is closed \n", port)
+		} else {
+			fmt.Printf("Port %s is open \n", port)
+		}
+	}
 }

